src/java.desktop/share/classes/javax/swing/plaf/basic/BasicSeparatorUI.java

```diff
@@ -25,12 +25,14 @@
 
 package javax.swing.plaf.basic;
 
-import javax.swing.*;
 import java.awt.Color;
 import java.awt.Dimension;
 import java.awt.Graphics;
 import java.awt.Insets;
 import java.awt.Rectangle;
```

Comment:
Neither `Insets`, nor `Rectangle` are used here.
```suggestion
```

Reply:
it was there even before this PR but anyway have removed...

Reply:
They were there, but why are there if they're unused?